contracts/lending/interfaces/INFTLending.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+/// @notice NFTLending Interface
+interface INFTLending {
```

Comment:
let's add a new view function that returns the nft address and the nft id given a loan id

function getNFTDetails(uint256 _loanId) returns (address nftAddress, uint256 tokenId)